backend/inspirehep/pidstore/errors.py

```diff
@@ -20,3 +24,18 @@ class MissingControlNumber(BaseInspirePidStoreError):
 
 class CNUMChanged(BaseInspirePidStoreError):
     pass
+
+
+class PIDError(BaseRestError):
+    pass
+
+
+class PIDAlreadyExistsError(PIDAlreadyExists, PIDError):
+    code = 400
+    description = "PIDAlreadyExists: pid_type:'{pid_type}', pid_value:'{pid_value}'."
+
+    def __init__(self, pid_type, pid_value, **kwargs):
+        super().__init__(pid_type=pid_type, pid_value=pid_value, **kwargs)
+        self.description = self.description.format(
+            pid_type=pid_type, pid_value=pid_value
```

Comment:
Didn't we decided to use fstrings?
`description = lambda: f"PIDAlreadyExists: pid_type:'{pid_type}', pid_value:'{pid_value}'."

and self.description=self.description()` or smthng like that?

Reply:
yes indeed, I just put the template as a static. Anyway changed it :)